options/locale/locale_en-US.ini

```diff
@@ -1412,6 +1412,7 @@ issues.filter_sort.fewestforks = Fewest forks
 issues.keyword_search_unavailable = Currently searching by keyword is not available. Please contact your site administrator.
 issues.action_open = Open
 issues.action_close = Close
+issues.action_delete_confirm = Confirm to delete all issues?
```

Comment:
Would prefer something more generic:

```ini
delete_confirm = Delete all items?
```

And likely move it to top-level translation.

Reply:
Done in 89604415524d389dd19977ad006501d25ccab2b3

But I still do not think it is a generic string.

And the "generic" problem is: without careful maintenance, it will become more and more messy. For example, why `rerun_all = Re-run all jobs`appears in the global scope? It isn't generic indeed.

Reply:
If a translation could reasonably appear in more than 1 scope, it should be global, but I guess the one you posted is likely to be single-use.

Overall, I think we should just get rid of scopes altogether in the translation, removing all duplicates in the process. Those scope have no benefit except encourage duplication.